swagger.yaml

```diff
@@ -49,6 +49,9 @@ securityDefinitions:
 paths:
   /population-types:
     get:
+      parameters:
+      - $ref: '#/parameters/page_limit'
+      - $ref: '#/parameters/offset'
```

Comment:
I can't see any params for this endpoint? Am I missing something or looking at the wrong place?

Reply:
We have a ticket to implement these that is being worked on at the moment - https://trello.com/c/ey4DIlBl/5896-implement-pagination-on-population-types-endpoint